### DISCUSSION TITLE
Help Sharing Music Folder in Samba, or ways to access wirelessly

### DISCUSSION THREAD

**Discussion by dinkenstein** [marcone/teslausb#784] [Q&A]
So the TeslaCam folder can be shared in Samba as read-only \\teslausb\TeslaCam
Is there an way to enable Samba for the Music folder with write access, to easily add to music to car wirelessly?

I tried to mess around with smb.conf but I could not locate the path to the Music folder. I do understand this can be achieved via Music server sync, but don't have a NAS server at the moment.

**Comment by KrzysztofHajdamowicz** [comment]
You can edit setup variables so everytime recordings are pushed from TeslaUSB to file server, music can be pulled from file server to TeslaUSB.

**Reply by dinkenstein** [reply]
I was aware of this function, but do not have a file server. 
would like to find an easy way to write to music folder via wifi. 

I tried editing smb.conf to add entry for music folder, unfortunately the music folder is not mounted to a path, and I'm not good with Linux at all😕

**Comment by marcone** [comment]
Exposing the music drive as a writeable Samba drive would require preventing access from the car while the drive is mounted via Samba, which may be tricky. It would probably be best to add something to the web interface to manage the music drive.

**Comment by dinkenstein** [comment]
Bummer, I was hoping for a quick samba share workaround. A web UI to upload songs would be a really cool feature though!
